spark/v3.1/spark/src/main/java/com/netease/arctic/spark/ArcticSparkCatalog.java

```diff
@@ -75,7 +74,7 @@
 import static com.netease.arctic.spark.SparkSQLProperties.USE_TIMESTAMP_WITHOUT_TIME_ZONE_IN_NEW_TABLES_DEFAULT;
 import static org.apache.iceberg.spark.SparkSQLProperties.HANDLE_TIMESTAMP_WITHOUT_TIMEZONE;
 
-public class ArcticSparkCatalog implements TableCatalog, SupportsNamespaces {
+public class ArcticSparkCatalog extends BaseCatalog {
```

Comment:
Why inherit this class instead of implementing the relevant interface? Are there any reusable methods needed?